app/presenters/track.rb

```diff
@@ -0,0 +1,45 @@
+module ExercismWeb
+  module Presenters
+    class Track
+
+      def initialize(track_id)
+        @track_id = track_id
+      end
+
+      def method_missing(method, *args)
+        if trackler_track.respond_to?(method)
+          trackler_track.send(method, *args)
+        else
+          super
+        end
+      end
+
+      def docs
+        track_docs = trackler_track.docs("/api/v1/tracks/%s/images/docs/img" % @track_id)
+
+        track_docs.each_pair do |topic_name, topic_content|
+          track_docs[topic_name] = if topic_content.present?
+                                     topic_content
+                                   else
+                                     default_topic_content(topic_name)
+                                   end
+        end
+
+        track_docs
+      end
+
+      private
+
+      def default_topic_content(topic_name)
```

Comment:
After thinking about this a bit I think that `fallback` makes more sense than `default` here. I'm going to go ahead and just make the change and then merge this.